build.sbt

```diff
@@ -5,4 +5,5 @@ version := "0.2"
 scalaVersion := "2.13.1"
 
 libraryDependencies += "com.softwaremill.sttp.client" %% "core" % "2.0.1"
-libraryDependencies += "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % "2.0.1"
\ No newline at end of file
+libraryDependencies += "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % "2.0.1"
```

Comment:
Move version to `val` and/or create Dependencies object in the project folder and manage dependencies there

Reply:
Done